templates/mellon.conf.epp

```diff
@@ -3,6 +3,10 @@
     AuthType "Mellon"
     MellonEnable auth
     MellonMergeEnvVars On
+<% if $facts['os']['name'] == 'Ubuntu' and Float($facts['os']['release']['major']) >= 20.04 { -%>
```

Comment:
Is there a reason to limit this change to just this ubuntu version?

Reply:
Yes, these parameters support only 0.16 version of mellon and above. Currently only Ubuntu 20.04 has 0.16 version of mellon. All older versions of Ubuntu and other distros like CentOS and RedHat have older versions of mellon where these parameters will crash configuration. So here I set: apply for Ubuntu 20.04 and other future versions of Ubuntu which are higher 20.04